### PR TITLE
Fix: #4466 - Add Release Notes for 1.32.1

### LINKED ISSUE #4466: Release Notes for 1.32.1
- Added CarPlay support for playlist. ([#3673](https://github.com/brave/brave-ios/3673))
- Added a new section in history for entries older than last month. ([#4040](https://github.com/brave/brave-ios/4040))
- Added support for sharing playlist items. ([#4191](https://github.com/brave/brave-ios/4191))
- Added an option to disable "Add to playlist" button in toolbar.  ([#4234](https://github.com/brave/brave-ios/4234))
- Implemented 'Bookmark all tabs' feature. ([#4132](https://github.com/brave/brave-ios/4132))
- Updated default browser notification to display device's iOS version. ([#3899](https://github.com/brave/brave-ios/3899))
- [Security] Fixed address bar spoofing via interstitial content as reported on HackerOne by dogeshark. ([#4396](https://github.com/brave/brave-ios/4396))
- Fixed app freeze when Rewards are enabled. ([#3941](https://github.com/brave/brave-ios/3941))
- Fixed favorites title when no favorites are listed. ([#4128](https://github.com/brave/brave-ios/4128))
- Fixed black screen on tabs when launching Brave app. ([#4190](https://github.com/brave/brave-ios/4190))

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

This pull request fixes #4466

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Test Plan:

NA

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [X] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.32.1](https://github.com/brave/brave-ios/releases/tag/v1.32.1)
+
+- Added CarPlay support for playlist. ([#3673](https://github.com/brave/brave-ios/3673))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: should capitalize `playlist` -> `Playlist` as we're referring to the feature itself in this instance.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+- Added a new section in history for entries older than last month. ([#4040](https://github.com/brave/brave-ios/4040))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: should add `browsing history` to be more specific. Just mentioning `history` could be anything without context.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+- Added support for sharing playlist items. ([#4191](https://github.com/brave/brave-ios/4191))
+- Added an option to disable "Add to playlist" button in toolbar.  ([#4234](https://github.com/brave/brave-ios/4234))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: there's a double space between the issue description and link.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+- Implemented 'Bookmark all tabs' feature. ([#4132](https://github.com/brave/brave-ios/4132))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: use `"` or the CHANGELOG will get formatted. We usually don't use the `formatting` but use "formatting".

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
...
+- Updated default browser notification to display device's iOS version. ([#3899](https://github.com/brave/brave-ios/3899))
+- [Security] Fixed address bar spoofing via interstitial content as reported on HackerOne by dogeshark. ([#4396](https://github.com/brave/brave-ios/4396))
+- Fixed app freeze when Rewards are enabled. ([#3941](https://github.com/brave/brave-ios/3941))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: something like the following would sound a bit better:

```
Fixed app freeze when enabling rewards.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,18 @@
...
+- Fixed favorites title when there are no favorites. ([#4128](https://github.com/brave/brave-ios/4128))
```
Comment (kjozwiak) [inline CHANGELOG.md]: hmmm maybe adding a bit more context? For example:

```
 - Removed the "Favorites" title in absence of any favorites under New Tab Page when focusing on URL bar
```

**Review by kjozwiak — CHANGES_REQUESTED** [review]

**Review by kjozwiak — APPROVED** [review]
LGTM 👍 Awesome work.

**Comment by kjozwiak** [comment]
@srirambv merged so I can get the ones for `1.32.2` created 👍 Mind updating https://github.com/brave/brave-ios/releases/tag/untagged-d376a299cb09f299baa8 whenever you get the chance? Thanks sir!

**Comment by srirambv** [comment]
@kjozwiak its done. Sorry for the delay in updating the releases

**Comment by kjozwiak** [comment]
> @kjozwiak its done. Sorry for the delay in updating the releases

no worries 👍 Thanks for getting the notes done! I'll get the `1.32.2` notes created which are really small so shouldn't take too long.

**Pull request merged**